templates/base/src/__tests__/_document.test.ts

```diff
@@ -0,0 +1,8 @@
+import { render } from '@testing-library/react';
+import MyDocument from '../pages/_document';
+
+describe('<MyDocument />', () => {
+  it('should render without errors', async () => {
+    expect(true).toBe(true);
```

Comment:
Check for component existence here.

Reply:
@vinodloha  Not Able to render it like a normal react based component with Testing library's render method. It shows error that a class component cant be called without new